src/utils/gameTimer.ts

```diff
@@ -0,0 +1,21 @@
+import type { GameStartType } from "..";
+import { TimerType } from "..";
+import { format } from "date-fns";
+
+export function frameToGameTimer(frame: number, gameStart: GameStartType): string {
```

Comment:
I think you can simplify this function type signature. If we're only converting numbers to strings essentially and doing formatting, we probably don't need to know about the entirety of the `GameStartType`. Really it only needs to know whether or not the game timer is decreasing and if so, what the starting timer was.

So you could probably just do something like:

```suggestion
export function frameToGameTimer(frame: number, startingTimer?: number): string {
```
If `startingTimer` is passed in, then assume the timer is decreasing, otherwise assume that the timer is increasing.

Reply:
I'm not a fan of this solution since we also have the option to have the timer disabled. I can add an extra parameter for timer type if you think that's a good compromise.

Reply:
By "timer disabled", is that different from `TimerType.NONE`?

This function still shouldn't depend on the entirety of the `GameStartType` object. So you can just pick the required fields instead.
```suggestion
export function frameToGameTimer(frame: number, options: Pick<GameStartType, "timerType" | "startingTimerSeconds">): string {
```